### PR TITLE
PI-1022: Upgrading frontend to 2.6.5.  Enabling the Cache by Cookie C…

### PR DESCRIPTION
**Pull request by jwineman**
…ard.

### CODE REVIEW

**Comment by jwineman** [comment]
Removed non en translations because they're way out of date and essentially useless.

File: view/adminhtml/web/config.js [hunk 1]
```diff
@@ -21,10 +21,10 @@
     "homePageCards": ["ApplyDefaultSettingsCard", "PluginSpecificCacheTagCard", "PurgeCacheCard"],
     "moreSettingsCards": {
         "container.moresettings.security": ["SecurityLevelCard", "WAFCard"],
-        "container.moresettings.performance": ["AlwaysOnlineCard", "ImageOptimizationCard", "DevelopmentModeCard"]
+        "container.moresettings.performance": ["BypassCacheByCookieCard", "AlwaysOnlineCard", "ImageOptimizationCard", "DevelopmentModeCard"]
```
Comment (thellimist) [inline view/adminhtml/web/config.js]: Do we want `BypassCacheByCookieCard` at the top? The order here is also the order of the cards to be rendered

**Comment by thellimist** [comment]
Fo easier debugging does it make sense to put compiled.js.map to the repository?

**Comment by jwineman** [comment]
idk maybe.  I figure being able to look at the redux logger actions is sufficient enough for most issues.  If you feel strongly about it let me know and I'll put it in.

**Comment by thellimist** [comment]
It's fine. I personally find it useful but I have no strong opinions.

**Pull request merged**